src/import-data/ImportDataDestination.test.ts

```diff
@@ -110,6 +122,57 @@ describe('ImportDataDestination', () => {
     }
   );
 
+  it.each([
+    {
+      importRequest: { type: 'pfb', url: new URL('https://service.prod.anvil.gi.ucsc.edu/path/to/file.pfb') },
+      shouldSelectExisting: false,
+    },
+    {
+      importRequest: { type: 'pfb', url: new URL('https://example.com/path/to/file.pfb') },
+      shouldSelectExisting: true,
+    },
+  ] as {
+    importRequest: ImportRequest;
+    shouldSelectExisting: boolean;
+  }[])('should disable start with an existing workspace option', async ({ importRequest, shouldSelectExisting }) => {
+    // Arrange
+    const user = userEvent.setup();
+    asMockedFn(canImportIntoWorkspace).mockImplementation(
+      (_importOptions: ImportOptions, workspace: WorkspaceWrapper): boolean => {
+        return workspace.workspace.name === 'allowed-workspace-not';
```

Comment:
Where does this "allowed-workspace-not" name come from?

Reply:
its basically there to return the fact that there is no matching protected workspace that can be imported into. there is likely a better way to do this check but my brain couldnt quite figure it out 😆

Reply:
Maybe just `return false`?

Reply:
that makes so much sense! morning sinus brain is failing